next.config.js

```diff
@@ -34,6 +35,40 @@ const nextConfig = {
 		// see https://styled-components.com/docs/tooling#babel-plugin for more info on the options.
 		styledComponents: true,
 	},
+	webpack: (
+		config,
+		{ buildId, dev, isServer, defaultLoaders, nextRuntime, webpack },
```

Comment:
`buildId`, `defaultLoaders`, `nextRuntime` and `webpack` seems unused

Reply:
true, removed :+1: